<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>734</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1004</width>
    <height>734</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Serial port terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="Port1">
      <property name="title">
       <string>Port 1</string>
      </property>
      <widget class="QComboBox" name="baudPort1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connectPort1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>570</y>
         <width>441</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="symPort1">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Symbol</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cyclePort1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>121</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Cyclical sending</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendPort1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="hexPort1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="intervalPort1">
        <property name="geometry">
         <rect>
          <x>282</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="clearOutputPort1">
        <property name="geometry">
         <rect>
          <x>349</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear Output</string>
        </property>
       </widget>
       <widget class="QLabel" name="mcsLabel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>mcs</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="disconnectPort1">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="settingsPort1">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
      <widget class="QComboBox" name="namePort1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>130</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>490</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="inputPort1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>441</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="outputPort1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>530</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="clearInputPort1">
       <property name="geometry">
        <rect>
         <x>379</x>
         <y>470</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Input</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addPort">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>80</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Add port</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Port2">
      <property name="title">
       <string>Port 2</string>
      </property>
      <widget class="QComboBox" name="baudPort2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connectPort2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>570</y>
         <width>441</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QRadioButton" name="symPort2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Symbol</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cyclePort2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>121</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Cyclical sending</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendPort2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="hexPort2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>84</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="intervalPort2">
        <property name="geometry">
         <rect>
          <x>282</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="clearOutputPort2">
        <property name="geometry">
         <rect>
          <x>349</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear Output</string>
        </property>
       </widget>
       <widget class="QLabel" name="mcsLabel_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>60</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>mcs</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="disconnectPort2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="settingsPort2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
      <widget class="QComboBox" name="namePort2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>72</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>130</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>490</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="inputPort2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>441</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="outputPort2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>530</y>
         <width>441</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QPushButton" name="clearInputPort2">
       <property name="geometry">
        <rect>
         <x>379</x>
         <y>470</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear Input</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delPort">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>80</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete port</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
